src/renderer/dx/DxRenderer.hpp

```diff
@@ -181,6 +184,7 @@ namespace Microsoft::Console::Render
         ::Microsoft::WRL::ComPtr<ID2D1SolidColorBrush> _d2dBrushForeground;
         ::Microsoft::WRL::ComPtr<ID2D1SolidColorBrush> _d2dBrushBackground;
         ::Microsoft::WRL::ComPtr<IDXGISwapChain1> _dxgiSwapChain;
+        HANDLE _swapChainFrameLatencyWaitableObject;

```

Comment:
This should probably be put in some sort of `wil::unique_handle`, just to be safe

Reply:
I would assume that the handle is owned by the swap chain, like a C++ reference that would be owned by another object, and that it shouldn't be deleted by our code, because the object is "created with the swap chain" and not by calling a method, but I'm not sure.
The docs don't really say whether to delete it or not.
We should probably wait for the expert.

Reply:
That's super weird that the docs don't call out explicitly closing that `HANDLE`, or even explicitly _not_ closing it. You might be right that it's not important - DX isn't really my area of expertise.

Reply:
I just found the samples from MS https://github.com/microsoftarchive/msdn-code-gallery-microsoft/tree/master/Official%20Windows%20Platform%20Sample/DirectX%20latency%20sample/%5BC%2B%2B%5D-DirectX%20latency%20sample/C%2B%2B : they use a raw HANDLE and they don't close it (in DeviceResources.h/cpp)